### ISSUE TITLE
Upgrade to ember-cli-babel@^6.0.0

### ISSUE THREAD

**Issue by Blackening999** [offirgolan/ember-cp-validations#520]
This addon is outdated with the recent Ember versions

```
ember-cli-pagination@3.0.2:
  version "3.0.2"
  resolved "https://registry.yarnpkg.com/ember-cli-pagination/-/ember-cli-pagination-3.0.2.tgz#9d51f2b6ad80b0b98729f0b18de8a2d02e4fc26b"
  dependencies:
    ember-cli-babel "^5.1.6"
    ember-cli-htmlbars "^1.1.1"
    ember-cli-version-checker "^1.1.5"
```

```
$> ember -v

ember-cli: 2.14.1
node: 8.2.1
os: darwin x64
```

**Comment by offirgolan** [comment]
Yeah sorry about that! I'll try to upgrade all the dependencies within the next week or so.

**Comment by Blackening999** [comment]
@offirgolan not a big deal :) just pinging everyone :) Thank you very much!

**Comment by Turbo87** [comment]
@Blackening999 I'd recommend opening pull requests and updating the dependencies 😉 

in this project I've already done that though (see https://github.com/offirgolan/ember-cp-validations/issues/521)

**Comment by Blackening999** [comment]
@Turbo87 I hate to ask people do something they're doing for free But I simply can't do that. I have about 30 dependencies with issues like that or more serious :(

And... you're awesome! Thank you for #521!
It's cool if you can simply bump 2 deps, for some libraries it's not that simple :\

**Comment by Turbo87** [comment]
> I have about 30 dependencies with issues like that

then open 30 PRs. that's what I'm currently doing...

**Comment by rwjblue** [comment]
>  But I simply can't afford to update every dependency I have :( I have about 30 dependencies with issues like that or more serious :(

If everyone did this, nothing would get done 😈 .

**Comment by offirgolan** [comment]
Resolved with #523. Released with `v3.4.0`.
